### DISCUSSION TITLE
Version locale

### DISCUSSION THREAD

**Discussion by Gamer000gaming** [Crownicles/Crownicles#2364] [Suggestions fermées]
Je propose de créer une version "locale" où tout est stocké sur le serveur discord et non sur le profil du joueur. Pour cette version il faudra des commandes du type admin pour ajouter ou retirer de l'argent à des joueurs.

**Comment by BastLast** [comment]
Hello, c'est vraiment pas la direction sur laquelle on s'oriente, si tu veux faire cela, il faut fork le projet.

**Reply by Gamer000gaming** [reply]
D'accord c'est ce que je vais faire
